localdev/config/executor/config.yaml

```diff
@@ -36,7 +36,7 @@ kubernetes:
   stuckTerminatingPodExpiry: 1m
   podDefaults:
     ingress:
-      hostnameSuffix: "svc"
+      hostnameSuffix: "localhost"
```

Comment:
Can I ask what the behavior of this is?

Like for example, say you have a pod on {pod-id}.{etc etc}.localhost, will that resolve?

Reply:
it may or may not, but was just changing to match e2e config

Reply:
I doubt it will resolve. But if it does, @kannon92 this might let you get the armada-jupyter stuff working?